### ISSUE TITLE
Exposing behave differently for exports and imports

### ISSUE THREAD

**Issue by mpizenberg** [avh4/elm-format#556]
Version: 0.8.0
The following code:
```elm
module ExposingNewLine exposing
    ( First
    , Second)

import Module exposing
    ( First
    , Second)
```
Gets transformed into:
```elm
module ExposingNewLine exposing
    ( First
    , Second
    )

import Module
    exposing
        ( First
        , Second
        )
```

**Comment by mdevlamynck** [comment]
I just realized going the other way around might be a better solution.
```elm
module ExposingNewLine exposing
    ( First
    , Second
    )

import Module exposing
    ( First
    , Second
    )
```
But this way it becomes unclear to me how to handle the where clause. What do you think?

**Comment by mpizenberg** [comment]
I've personally never seen the `where` clause. It might be for effect modules only, that are not allowed anymore outside of core and exploration. The new 0.8 formatting is supposed to have switched to exposing being on the same line to my knowledge.

**Comment by mdevlamynck** [comment]
I've only found out about it while digging in the elm-format code and tests. But if there are plans to use elm-format on core code someday it should be supported. Plus it's currently handled, unifying formatting behavior shouldn't break existing feature IMHO.

**Comment by avh4** [comment]
Yes, the way `module` lines are currently formatted is correct.  If anything is to change here, it will need to be the `import` lines.

Though now that I look at it closer, I'm not sure that there should be any change.  The thought for imports was that

```
import X.ModuleName
    as ModuleName
    exposing ( A, B )
```

is that it lets you easily scan the separate clauses (`as`, `exposing`), so I'm not sure putting `exposing` at the end would be better, especially given that `as` is commonly used, and is important to be prominently visible when scanning the code.

The reason `module` lines were changed was at specific request from Evan, and typically when looking at them, you are always looking for the exposings (and as noted, `where` clauses are extremely rare).

**Comment by mdevlamynck** [comment]
For reference elm-format currently formats
```elm
import X.ModuleName
    as ModuleName
    exposing ( A, B )

-- into

import X.ModuleName as ModuleName exposing (A, B)
```
and
```elm
import X.ModuleName
    as ModuleName
    exposing
        ( A
        , B
        )

-- into

import X.ModuleName as ModuleName
    exposing
        ( A
        , B
        )
```

@avh4 I'm not sure I understand your reply. Can you expand a bit on your thoughts about `import`?

**Comment by avh4** [comment]
Sure, I guess I'm wondering about the reasons we're considering making a change to the current behavior.  The proposed reason is that `import` and `module` lines are formatted differently.  Is that a good reason?  True, they both have an `exposing` clause, but apart from that they have very different meanings.  Are there additional reasons that changing the formatting of `import` makes sense?  Or are there additional reasons that having `import` and `module` lines be formatted the same would be beneficial?

If we are considering changes to how `import` lines are formatted, then I think we should be considering what changes will improve the readability of `import` lines.  I guess it's not clear to me what the best change would be, and I think we should have some more convincing reasons before making a change.

**Comment by mpizenberg** [comment]
Fair enough, I guess I have a personal preference for the new exports formatting and was wondering if for consistency the imports should change also. I don't have strong arguments to support my taste, it just felt "odd" when I saw both ways at the top of my file.
